Defs/Stats/Stats_NightVision.xml

```diff
@@ -66,7 +66,7 @@
 
 	<StatDef ParentName="EquipmentNightVisionBase">
 		<defName>NightVisionEfficiency_Weapon</defName>
-		<label>night vision efficiency</label>
+		<label>weapon night vision efficiency</label>
```

Comment:
Is there any reason to specify in the label? I don't believe it's possible for the various types of night vision efficiencies to appear together on the same panel, is it?

Reply:
![image](https://github.com/user-attachments/assets/77aecb9b-3960-4e78-962f-5b43cbe8dcf3)

The stat explanation for the combined night vision stat now displays all 3 sources. And is also where one of the new translation keys is used. The other is on the currently unused statPart that selects the smallest value that's available instead.

Reply:
Ah, I see. Thoughts on changing the format to `night vision efficiency (weapon)`?

Reply:
I'm not sure if that's an improvement over the current one, it's probably better when viewed side-by-side like in the screenshot, but I dunno if that's the case for individual items.